src/Features/Onboarding.php

```diff
@@ -418,24 +419,109 @@ public static function update_help_tab() {
 			}
 
 			$screen->remove_help_tab( 'woocommerce_onboard_tab' );
-			$help_tab['content'] = '<h2>' . __( 'Setup wizard', 'woocommerce-admin' ) . '</h2>' .
-				'<p>' . __( 'If you need to access the setup wizard again, please click on the button below.', 'woocommerce-admin' ) . '</p>' .
-				'<p><a href="' . wc_admin_url( '&reset_profiler=1' ) . '" class="button button-primary">' . __( 'Setup wizard', 'woocommerce-admin' ) . '</a></p>';
+
+			$task_list_hidden = get_option( 'woocommerce_task_list_hidden', 'no' );
+			$onboarding_data  = get_option( 'wc_onboarding_profile', array() );
+			$is_completed     = isset( $onboarding_data['completed'] ) && true === $onboarding_data['completed'];
+			$is_skipped       = isset( $onboarding_data['skipped'] ) && true === $onboarding_data['skipped'];
+			$is_enabled       = $is_completed || $is_skipped ? false : true;
+
+			$help_tab['content'] = '<h2>' . __( 'WooCommerce Onboarding', 'woocommerce-admin' ) . '</h2>';
+
+			$help_tab['content'] .= '<h3>' . __( 'Profile Setup Wizard', 'woocommerce-admin' ) . '</h3>';
+			$help_tab['content'] .= '<p>' . __( 'If you need to enable or disable the setup wizard again, please click on the button below.', 'woocommerce-admin' ) . '</p>'.
+			( $is_enabled
+				? '<p><a href="' . wc_admin_url( '&reset_profiler=0' ) . '" class="button button-primary">' . __( 'Disable', 'woocommerce-admin' ) . '</a></p>'
+				: '<p><a href="' . wc_admin_url( '&reset_profiler=1' ) . '" class="button button-primary">' . __( 'Enable', 'woocommerce-admin' ) . '</a></p>'
+			);
+
+			$help_tab['content'] .= '<h3>' . __( 'Task List', 'woocommerce-admin' ) . '</h3>';
+			$help_tab['content'] .= '<p>' . __( 'If you need to enable or disable the task list, please click on the button below.', 'woocommerce-admin' ) . '</p>' .
+			( 'yes' === $task_list_hidden
+				? '<p><a href="' . wc_admin_url( '&reset_task_list=1' ) . '" class="button button-primary">' . __( 'Enable', 'woocommerce-admin' ) . '</a></p>'
+				: '<p><a href="' . wc_admin_url( '&reset_task_list=0' ) . '" class="button button-primary">' . __( 'Disable', 'woocommerce-admin' ) . '</a></p>'
+			);
+
+			$help_tab['content'] .= '<h3>' . __( 'Calypso', 'woocommerce-admin' ) . '</h3>';
```

Comment:
Since store owners will also have access to this tab, should we hide this behind a dev flag or do you think it will be useful to them as well?

If it is useful to them, I think changing this to `WordPress.com` may be more recognizable to store owners.

Reply:
Great point. I think it probably makes sense to hide behind a dev flag.

Reply:
https://github.com/woocommerce/woocommerce-admin/pull/2863/commits/52ef43e6054181fd540f56f12d5f12d62a2cc2ba